db/migrate/20180701234758_add_role_to_users.rb

```diff
@@ -0,0 +1,24 @@
+class AddRoleToUsers < ActiveRecord::Migration
+
+  # 0 = nobody, 100 = admin.  See enum in models/user.rb
+  # There must be a better way to do this
+
+  @@user = 10
+  @@admin = 100
+
+  def up
+    add_column :users, :role, :integer, null: false, default: @@user
+    User.connection.execute("UPDATE users SET role = #{@@admin} WHERE admin = true")
```

Comment:
THANK YOU. :) I was watching for this.  ;)  Of course, you can do it more elegantly with `User.where(admin: true).update_all(role: User.roles[:admin])` ... but still!

Reply:
I tried rewriting the sql commands as you suggested but couldn't come up with something that I had confidence in. So I've left these alone - they will only be run once per installation anyhow.